tests/unit/acquisition/function/test_greedy_batch.py

```diff
@@ -168,6 +176,137 @@ def test_lipschitz_penalizers_raises_for_invalid_pending_points_shape(
     penalizer: Callable[..., PenalizationFunction],
 ) -> None:
     best = tf.constant([0], dtype=tf.float64)
-    lipshitz_constant = tf.constant([1], dtype=tf.float64)
+    lipschitz_constant = tf.constant([1], dtype=tf.float64)
     with pytest.raises(TF_DEBUGGING_ERROR_TYPES):
-        soft_local_penalizer(QuadraticMeanAndRBFKernel(), pending_points, lipshitz_constant, best)
+        soft_local_penalizer(QuadraticMeanAndRBFKernel(), pending_points, lipschitz_constant, best)
+
+
+def test_fantasized_expected_improvement_builder_raises_for_invalid_num_samples() -> None:
+    with pytest.raises(tf.errors.InvalidArgumentError):
+        FantasizeAcquisitionFunction(ExpectedImprovement().using("OBJECTIVE"), "notKB")
+
+
+def test_fantasized_expected_improvement_builder_raises_for_invalid_model() -> None:
```

Comment:
Maybe add an integration test too?

Reply:
if that's OK I'll leave it for another PR? there's no integration test for LP either so we might do both at the same time. The 2 notebooks act as integration tests for now

Reply:
yes that's fine (the notebook is enough proof for now that integration isn't totally broken)